features/product/managing_products/sorting_products.feature

```diff
@@ -63,3 +64,22 @@ Feature: Sorting listed products
         And I sort the products descending by name
         Then I should see 3 products in the list
         And the first product on the list should have name "Szałowy Mops"
+
+    @ui
+    Scenario: Missing translations are sorted as first when sorting by name ascending
+        When I change my locale to "Polish"
+        And I browse products
+        And the products are already sorted ascending by name
+        Then I should see 3 products in the list
+        And the first product on the list should have name "brakujące tłumaczenie"
+        And the last product on the list should have name "Ekstremalny Mops"
+
+    @ui
+    Scenario: Missing translation are sorted as last when sorting by name descending
+        When I change my locale to "Polish"
+        And I browse products
+        And the products are already sorted ascending by name
+        And I sort the products descending by name
+        Then I should see 3 products in the list
+        And the first product on the list should have name "Ekstremalny Mops"
+        And the last product on the list should have name "brakujące tłumaczenie"
```

Comment:
We should probably not check the exact text, but rather check is there a test data attribute `data-test-missing-translation` so it would not fail if the translation is changed in Crowdin :)